spec/features/review/user_views_reviews_spec.rb

```diff
@@ -6,14 +6,24 @@
   so I can see just how bad the restaurant is
 ) do
 
-  scenario "user views all reviews for a restaurant" do
+  scenario "user views all reviews for a restaurant on their appropriate
+    pages" do
     restaurant = FactoryGirl.create(:restaurant)
-    review_one = FactoryGirl.create(:review, restaurant: restaurant)
-    review_two = FactoryGirl.create(:review, restaurant: restaurant)
+    next_page_review = FactoryGirl.create(
+    :review, restaurant: restaurant, body: "this review is on the next page"
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.